hpat/tests/test_series.py

```diff
@@ -545,7 +545,7 @@ def test_series_op7(self):
             pd.testing.assert_series_equal(hpat_func(A, B), test_impl(A, B), check_names=False)
 
     def test_series_op8(self):
-        comparison_methods = ('lt', 'gt', 'le', 'ge', 'ne', 'eq')
```

Comment:
I would propose to do not remove this test but add integers algo branch into `ne()` method in this PR